### DISCUSSION TITLE
How do I put a breakpoint on the left and get the line number

### DISCUSSION THREAD

**Discussion by mingyaodagege** [icsharpcode/AvalonEdit#349] [General]
How do I put a breakpoint on the left and get the line number

**Comment by siegfriedpammer** [comment]
There is a working implementation of breakpoints in SharpDevelop. A starting point would be the implementation of the bookmarks margin: https://github.com/icsharpcode/SharpDevelop/blob/master/src/AddIns/DisplayBindings/AvalonEdit.AddIn/Src/IconBarMargin.cs

Hope this helps!
